### DISCUSSION TITLE
Installation for Windows

### DISCUSSION THREAD

**Discussion by Stofa14** [xtekky/gpt4free#2000] [General]
There is no g4f.ex file. The Zip file named "gpt4free-main" and it does not contain executable file "g4f.exe" to run the setup. Is there another Zip-file?

**Comment by Parthsadaria** [comment]
https://github.com/xtekky/gpt4free/releases/download/0.3.1.7/g4f.exe.zip 
here,

**Reply by Baptiste-Boin** [reply]
for Mac please

**Comment by Parthsadaria** [comment]
run the exe fine then go to http://127.0.0.1:8080/chat/ on ur browser
